src/Datadog.Trace/Tracer.cs

```diff
@@ -253,6 +261,8 @@ public static Tracer Instance
 
         internal IDogStatsd Statsd { get; private set; }
 
+        internal IOutboundHttpConvention OutboundHttpConvention { get; }
```

Comment:
I assume that in the end tracer has a single property for the semantic conventions instead of one for outbound HTTP a separate one for DB, etc. Right now it is only the first "target", correct?

Reply:
In other words, what are your plans for it to grow?

Reply:
**Option 1**. Have multiple fields
**Option 2**. Have a facade with multiple sematic conventions implementations

I would go with Option 1 and when we start having 3 interfaces then we should probably refactor it to Option 2.
I just do not want to create a single gigantic interface.

Reply:
SGTM